includes/post-hooks.php

```
@@ -578,45 +572,54 @@ function push_story_update( int $post_id, string $story_id, array $update_data,
-
 	if ( $date_changed ) {
-		$dates                     = calculate_story_dates( $post->post_date );
-		$update_data['start_date'] = $dates['start_date'];
-		$update_data['end_date']   = $dates['end_date'];
-		$update_data['weekdays']   = $dates['weekdays'];
+		return build_full_story_update( $post->post_title, $post->post_date );
 	}
 
-	return $update_data;
+	return array( 'title' => $post->post_title );
+}
```

PHPMD (2.15.0)

[error] The parameter $post_before is not named in camelCase. (undefined)

(CamelCaseParameterName)